.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
     -   id: reorder-python-imports
         args: [--py3-plus]
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.950
+    rev: v0.981
```

Comment:
Bumping this to 0.981 because we need this fix: https://github.com/python/mypy/pull/13398